src/understand-tidb/plugin.md

```diff
@@ -1 +1,63 @@
 # Plugin
+
+The plugin API allows TiDB to be extended with new features such as audit logging or IP allow/deny listing.
+
+Sample code is provided for a basic audit logging plugin at [`./tidb/plugin/conn_ip_example/`](https://github.com/pingcap/tidb/tree/master/plugin/conn_ip_example). For an example on compiling TiDB and this plugin:
+
+```bash
+plugin="conn_ip_example"
+cd cmd/pluginpkg
+go install
+cd ../../plugin/$plugin
+pluginpkg -pkg-dir . -out-dir .
+cd ../..
+ 
+./bin/tidb-server -plugin-dir /path/to/tidb/plugin/$plugin -plugin-load $plugin-1
```

Comment:
```suggestion
./bin/tidb-server -plugin-dir plugin/$plugin -plugin-load $plugin-1
```

with previous command, we already assume under the root path of tidb repo.

Reply:
I've incorporated this, thanks!